mod/etherpad/Dockerfile

```diff
@@ -3,10 +3,12 @@ FROM etherpad/etherpad:1.8.13
 USER root
 
 RUN apt-get update \
-    && apt-get install -y git
+    && apt-get install -y git curl
 
 USER etherpad
 
+ENV TAG v2.3-alpha-4
```

Comment:
Should this be 'beta' rather than 'alpha'?

Reply:
ah, yeps, absolutely!
I've pushed another commit to `develop`: https://github.com/bigbluebutton/docker/commit/8429a3b5115e9212b6716d0184d0374499cf7478